stravalib/protocol.py

```diff
@@ -80,8 +80,11 @@ def __init__(
             self.rsession = requests.Session()
 
         if rate_limiter is None:
-            # Make it a dummy function, so we don't have to check if it's defined before
-            # calling it later
+            # Make it a dummy function, so we don't have to check if it's
+            # defined before calling it later
+
+            # TODO: This violates E731 in PEP 8 - are we ok with that?
```

Comment:
I added a small commit to address this

Reply:
perfect -thanks! Jonatan -your fix looks like this

`self.rate_limiter = rate_limiter or (lambda _: None)` 
which also allowed you to remove another line of code. 

BUT why did you define the empty lamba function as `_` vs just using `(lambda: None)` ?
i'm just curious. i don't use lambda functions a lot but do understand it allows us to create an empty object here . and the or would return whatever has some value in this case OR the empty function.

many thanks!

Reply:
The `_` allows us to assign as `rate_limiter` an (anonymous) function that accepts a single (anonymous) positional argument. Without that argument (i.e., using `lambda : None`), the code using this will try to call that lambda _with_ an argument and cause a TypeError (or fail static type checks otherwise).

To me, the `_` signifies that the argument isn't used. We can also write `lambda x: None` (as in the original code) if that makes more sense to you (although that may raise a PEP warning about declared but unused variables).